owtf/models/user.py

```diff
@@ -13,7 +13,7 @@ class User(Model):
     __tablename__ = "users"
 
     id = Column(Integer, primary_key=True, autoincrement=True)
-    name = Column(Unicode(255))
+    name = Column(Unicode(255), nullable=False, unique=True)
```

Comment:
are you sure you want this set to `unique=True`? People have same names all the time xD

Reply:
name here refers to the unique username/ unique handle. It is not like the first or last name of the user. It is important to keep it unique to allow login by both username or email.

Reply:
ah perfect